content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -245,4 +250,5 @@ When the necessary information has been entered, click :guilabel:`Create Invoice
 then be sent to the customer for payment.
```

Comment:
I can't suggest any thing on a "greyed out" line, but in line 249, you say "click 'Create Invoice'" but the image shows a button reading: "Create Draft" -- it might be best to double check the accuracy of this button during your revisions, so as to avoid any confusion with the reader. Thanks!